src/main/java/rs/teslaris/core/configuration/BeanConfiguration.java

```diff
@@ -76,13 +87,38 @@ public void addCorsMappings(@NotNull CorsRegistry registry) {
     @Bean
     public MessageSource messageSource() {
         var messageSource = new ReloadableResourceBundleMessageSource();
-        messageSource.setBasename("classpath:internationalization/messages");
+
+        String baseName = resolveValidMessageSourceBaseName();
+        messageSource.setBasename(baseName);
```

Comment:
```suggestion
        messageSource.setBasenames(baseName, "classpath:internationalization/messages");
```

This allows specifying changes only for a specific filepath, while retaining the internal source as a fallback. It's useful when adding new messages to the messageSource.